### DISCUSSION TITLE
Old firmware

### DISCUSSION THREAD

**Discussion by tismofied** [gtxaspec/wz_mini_hacks#737] [Q&A] [answered]
I am still on the old RTSP demo firmware that was released by Wyze before they pulled it out of their website.
The camera itself is still on firmware **4.61.0.3** because I didn't want the RTSP firmware to get nuked. The current up-to-date firmware of the V3 model is : **4.36.11.8391**
I found out about Wyze mini hacks RTSP here and I was wondering if I should try my luck with it.
I am currently using the camera with frigate using Scrypted Wyze plugin and I am just wondering I guess If I should use a more recent RTSP firmware like the one offered here?
I still have about 6 other Wyze v3s sealed in box which I got during a Black Friday sale a year ago.

**Comment by Bob-AOL** [comment, marked as answer]
I'm still running the Wyze Beta 4.61.0.3 as well and have no issues with it. I only use wz_mini for the USB/ETH/POE. If it ain't broke. 

And wz_mini doesn't play nice past the regular OEM 4.36.9.139 anyway.

You may want to put your effort toward the USB/Ethernet feature with a UCTRONICS Mini USB to POE/Ethernet adapter if you already haven't. Well worth it.

**Reply by tismofied** [reply]
I didn't even know this was a thing. Thank you for bringing it to my attention. 
I just read through some of your posts on Wyzeforums and you seem to know more about Wyze v3 and I was just wondering if you know of a good way to power up 6 v3 cameras from one source. The reason I am asking is i would like to mount the cameras to each corner of the house and have the connections meet in the attic. Do you know of a way?

**Reply by Bob-AOL** [reply]
Good morning. (I just happen to be up early today. LOL)

The solution (and requirement) would be a POE switch if you are considering the wz_mini USB/ETH/POE feature. The switch would supply both power and ethernet connectivity to the cameras and connect to your router using just a single ethernet run. 

The only other requirement would be AC power for the switch itself.

**Reply by tismofied** [reply]
Good morning.
 I do have a POE switch I am using right now. Just have to work on running cat6 cables down to it lol. I have my network rack in the basement. 
Any particular UCTRONICS USB/ETH/POE  adaptor you recommend? a link perhaps?
Thank you so much for all your hard work on getting those pdfs done.  Without them, I don't think I'll be able to figure things out easily.

**Reply by Bob-AOL** [reply]
https://www.amazon.com/UCTRONICS-Upgraded-Raspberry-Chromecast-Compliant/dp/B0B9ZLVGJ4/ref=asc_df_B0B9ZLVGJ4/?tag=hyprod-20&linkCode=df0&hvadid=647242569010&hvpos=&hvnetw=g&hvrand=10418268565419689985&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9015610&hvtargid=pla-2018972422957&psc=1&mcid=0536c7f4fb4c3e5eacc6dc924311a8cf

**Reply by Bob-AOL** [reply]
You could put another switch in the attic and only need a single ethernet run to the basement. A little FE/100Mbps cheapie would do the trick if it has adequate environmental tolerance specs.

And fwiw if weatherization is required for the UCTRONICS adapters, I just use single gang or 4" weatherproof boxes from Lowes, Home Depot, or wherever. You can even use the magnet in the cam base to just mount them directly on the enclosure, though I do use one of the cover screws just to be safe.

Good luck with it.
